src/javafundamentals/interfaces/Ex01/CarsForSale.java

```diff
@@ -0,0 +1,13 @@
+package javafundamentals.interfaces.Ex01;
+
+public class CarsForSale implements SalesMethods{
```

Comment:
If this class represent a single car for sale then it should be named `CarForSale` instead of `CarsForSale`

Reply:
okay, that makes sense! thanks!